Updating passkeys promo blog (#2947)
FusionAuth/fusionauth-site: astro/src/content/blog/why-passkeys-matter.mdx

---
publish_date: 2024-03-12
title: Why Passkeys Matter
description: The world needs a better solution than passswords. Passkeys improve security while also being easier to use.
image: /img/blogs/why-passkeys/why-passkeys-matter.png
categories: Education
authors: Brad McCarty
tags: authentication, passkeys
excerpt_separator: "{/* more */}"
---
24 billion. That's how many passwords hackers [exposed](https://us.norton.com/blog/privacy/password-statistics#:~:text=Before%20diving%20into%20the%20nitty,%2C%20weak%2C%20or%20reused%20passwords.) in a single year. That's a staggering enough figure, and even more sobering when you realize that it gets bigger each year. The fact is that passwords are the worst combination of user experience and security. The best ones are difficult to remember, changed often, and never reused. It's no wonder that nearly 85 percent of people [recycle passwords](https://bitwarden.com/blog/a-closer-look-at-password-statistics/#:~:text=Here%20are%20some%20of%20findings,and%20names%20of%20loved%20ones).

> We need something better. Fortunately, [passkeys](/guides/what-is-a-passkey) are here as the answer. They're the replacement for passwords. As unlikely as the combination may seem, passkeys enhance security without sacrificing user experience.

Reviewer: If you're looking to be more positive.

```suggestion
We need something better. Fortunately, [passkeys](/guides/what-is-a-passkey) are here as the answer. They're the replacement for passwords. As unlikely as the combination may seem, passkeys enhance security while making the user experience better.
```

## The Limitations of Passwords

If everyone followed best practices for passwords, we wouldn't need to have this conversation. Using only capital and lowercase letters, there are over 218 trillion possible password combinations. When you add numbers and symbols, that number increases exponentially. Seems secure enough, right?

> But people aren't very good at creating passwords. Upwards of 13 percent of Americans [use the same password](https://explodingtopics.com/blog/password-stats) for every account. Only 37 percent even bother using two-factor authentication. These behaviors make for insecure logins that are vulnerable from their creation.

Reviewer: If you're looking to add a little humor.

```suggestion
But people aren't very good at creating passwords. Upwards of 13 percent of Americans [use the same password](https://explodingtopics.com/blog/password-stats) for every account. Only 37 percent even bother using two-factor authentication. There are also those few that still write their passwords on sticky notes and put it on their monitor. These behaviors make for insecure logins that are vulnerable from their creation.
```

## The Rise of Passkeys

Passkeys evolved out of the ideas behind public-key cryptography. This technology, first discussed in the 1970s, uses two keys. The public key is shared out in the open. The private key remains secure and confidential. The user can only gain access when both keys are present.

Fast forward to 2022 and big names in technology (Microsoft, Google, and Apple) joined up to unveil their support for passkeys on World Password Day. Now the challenge is getting the rest of the world to follow suit.

## Passkeys in Action

If you've used Windows Hello or Apple's FaceID, you've probably used passkeys. That PIN, fingerprint, or facial scan proves your identity. Once that's done, you're then given access to the public and private key pair to unlock the app in question.

> Let's get specific. Say that you are signing up for access to a website. You used to input a username and password combination that you'd have to remember. With passkeys, that same website says "this person is logged in to a secure device, and we can trust that they are who they say they are." Instead of using a username and password combination, you scan your fingerprint, face, or use a PIN to gain access to the site moving forward.

Reviewer: This is correct but seems a bit weird for me to read.  Just a suggestion.

```suggestion
Let's get specific. Say that you are signing up for access to a website. Typically, you would input a username and password combination that you'd have to remember. With passkeys, that same website says "this person is logged in to a secure device, and we can trust that they are who they say they are." Instead of using a username and password combination, you scan your fingerprint, face, or use a PIN to gain access to the site moving forward.
```

> Let's get specific. Say that you are signing up for access to a website. You used to input a username and password combination that you'd have to remember. With passkeys, that same website says "this person is logged in to a secure device, and we can trust that they are who they say they are." Instead of using a username and password combination, you scan your fingerprint, face, or use a PIN to gain access to the site moving forward.

Reviewer: " With passkeys, that same website says "this person is logged in to a secure device, and we can trust that they are who they say they are." 

This is a pretty big leap for the user to understand. I think there needs to be a little more detail like the user also needs to give permission to the website and that sort of thing.  

This is from our guide.

"
Passkeys are generated randomly and stored on the user's device. When logging into an account that requires a passkey, the user presents the passkey, which is then verified by the server. The server compares the passkey to stored templates and checks if it's valid. If it is, the user is authorized to log in. The only time the server has access to the passkey is during[ authentication](https://fusionauth.io/articles/authentication/).
"

Linking to something like https://fusionauth.io/guides/what-is-a-passkey could be enough too.
